research/gpu-exploration-app/README.md

```diff
@@ -0,0 +1,52 @@
+# ExampleGpuExploration
+
+This the a example for GPU exploration and a test for benchmarking results.
+
+## Documentation
+
+See [here](https://github.com/zkmopro/mopro/blob/main/mopro-core/gpu_explorations/README.md)
+
+## Usage
+
+> please install `mopro-cli` by following the instructions [here](https://github.com/zkmopro/mopro/tree/main/mopro-cli)
+
+### Prepare
+
+`mopro prepare`
+
+### Build
+
+`mopro build --platform ios`
+
+### Test
+
+`mopro test`
+
+The result would be shown as the following and in the folder `core/benchmarks/results/`
```

Comment:
The `mopro test` doesn't work for me.
maybe with clearer instructions how to reproduce this?

errors:
```sh
error[E0277]: the trait bound `ark_ff::fields::models::Fp256<FrParameters>: ark_ff::PrimeField` is not satisfied
  --> /Users/zhengyawen/.cargo/git/checkouts/mopro-471ecc433f4d4aff/371bfe5/mopro-core/src/middleware/gpu_explorations/arkworks_pippenger.rs:9:1
   |
9  | / pub fn benchmark_msm<I>(
10 | |     instances: I,
11 | |     iterations: u32,
12 | | ) -> Result<Vec<Duration>, preprocess::HarnessError>
13 | | where
14 | |     I: Iterator<Item = preprocess::Instance>,
   | |_____________________________________________^ the trait `ark_ff::PrimeField` is not implemented for `ark_ff::fields::models::Fp256<FrParameters>`
...
```

Reply:
well... this error seems like the cargo did not patch for arkworks version, but I think the Cargo.toml in this folder should have solved this :(

https://github.com/zkmopro/mopro/blob/8cf02da863f1fbe85f09e79964e181ff600afaea/research/gpu-exploration-app/core/Cargo.toml#L30-L35